SharperUniverse/Core/Builder/SystemBuilder.cs

```diff
@@ -48,7 +48,8 @@ internal SystemBuilder(GameRunner game, Dictionary<string, Type> commands)
                 foreach (var parameter in systemConstructor.GetParameters())
                 {
                     if (parameter.ParameterType != typeof(GameRunner) &&
```

Comment:
This line needs to be removed based on your changes. It could easily cause a breakage later down the line.